src/meili/routing.cc

```diff
@@ -543,6 +543,11 @@ find_shortest_path(baldr::GraphReader& reader,
             turn_cost += turn_cost_table[0];
           }
 
+          auto opp_edgeid = reader.GetOpposingEdgeId(origin_edge.id);
+          const baldr::GraphTile* endtile =
+              directededge->leaves_tile() ? reader.GetGraphTile(directededge->endnode()) : tile;
+          auto opp_edge = reader.directededge(opp_edgeid, endtile);
+
           // All destinations on this origin edge
           for (const auto other_dest : edge_dests[origin_edge.id]) {
```

Comment:
This be a const-reference? i.e.,

```c++
for (const auto& other_dest : edge_dests[origin_edge.id]) {
```

Reply:
it is a primitive type, do the copying is reasonably faster than use &, the auto key world is hiding what it is.

Reply:
it is a primitive type, just do copy is more efficient

Reply:
Ah, ok, I thought this was pointing to a container. nevermind!